api/app/signals/apps/feedback/models.py

```diff
@@ -0,0 +1,51 @@
+import secrets
+import uuid
+from datetime import timedelta
+
+from django.contrib.gis.db import models
+from django.utils import timezone
+
+from signals.apps.feedback.managers import FeedbackManager
+from signals.apps.signals.models import Signal
+
+FEEDBACK_EXPECTED_WITHIN_N_DAYS = 14  # move to general settings file
+
+
+def generate_token():
+    """Use secrets module of Python to generate a UUID object."""
+    return uuid.UUID(hex=secrets.token_hex(16))
+
+
+class StandardAnswer(models.Model):
+    is_visible = models.BooleanField(default=True)
+    is_satisfied = models.BooleanField(default=True)
+    text = models.TextField(max_length=1000)
+
+
+class Feedback(models.Model):
+    # Bookkeeping
+    token = models.UUIDField(db_index=True, primary_key=True, default=generate_token)
```

Comment:
Why use a function the function `generate_token` instead of `uuid.uuid4`?

Reply:
more secure random number generator

Reply:
Better random number generator in `secrets` standard lib module.